src/Collector/AbstractCollector.php

```diff
@@ -15,19 +17,31 @@ abstract class AbstractCollector implements CollectorInterface, \Serializable
      */
     protected $data;
 
+    public function __serialize()
+    {
+        return serialize($this->data);
+    }
+
     /**
-     * @see \Serializable
+     * @see Serializable
+     * @deprecated
```

Comment:
Marking a public API functionality as deprecated [requires a new minor version](https://semver.org/#spec-item-7).

Some more informations would be good, like:
```suggestion
     * @deprecated since 2.3.0, this method will be removed in version 3.0.0 of this component.
     *             {@see Serializable} as alternative
```
…or something similar.